src/Common/tests/Tests/System/Runtime/InteropServices/StringBufferTests.cs

```diff
@@ -0,0 +1,584 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace Tests.System.Runtime.InteropServices
+{
+    public class StringBufferTests
+    {
+        const string testString = ".NET Core is awesome.";
+
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
Is that possible? Also, we were doing some weird things with tests- do you know if we are building platform specific test assemblies yet for CI?